projects/step-core/src/lib/modules/dynamic-forms/components/dynamic-textfield/dynamic-textfield.component.html

```diff
@@ -14,14 +14,26 @@
 <ng-template #constantValueTemplate>
   <step-form-field [class.ng-invalid]="!!_ngControl.invalid" [class.with-tooltip]="tooltip">
     <step-label [attr.show-required-marker]="showRequiredAsterisk || null" *ngIf="label">{{ label }}</step-label>
-    <input
-      [type]="isNumber ? 'number' : 'text'"
-      [required]="showRequiredAsterisk"
```

Comment:
this part is no longer present it seems, do we need it?

Reply:
Input validity should depends on validator that is assigned to formControl.
It's not straightforward, that we have a component that implements `ControlValueAccessor` plus it has it's own validation internal logic.
Plus the attributes name `showRequiredAsterisk` informs that it's more about visual view.

Reply:
yea I guess at the time I put this we didnt have the ```[attr.show-required-marker]="showRequiredAsterisk || null"``` part so I put the required part in order for the asterisk to be shown